dlpy/applications/efficientnet.py

```diff
@@ -0,0 +1,292 @@
+        data is set after applying scaling and subtracting the specified offsets.
+        Default: (255*0.406, 255*0.456, 255*0.485)
+    norm_stds : double or iter-of-doubles, optional
+        Specifies a standard deviation for each channel in the input data.
+        The final input data is normalized with specified means and standard deviations.
+        Default: (255*0.225, 255*0.224, 255*0.229)
+    random_flip : string, optional
+        Specifies how to flip the data in the input layer when image data is
+        used. Approximately half of the input data is subject to flipping.
+        Valid Values: 'h', 'hv', 'v', 'none'
+    random_crop : string, optional
+        Specifies how to crop the data in the input layer when image data is
+        used. Images are cropped to the values that are specified in the width
+        and height parameters. Only the images with one or both dimensions
+        that are larger than those sizes are cropped.
+        Valid Values: 'none', 'unique', 'randomresized', 'resizethencrop'
+    random_mutation : string, optional
+        Specifies how to apply data augmentations/mutations to the data in the input layer.
+        Valid Values: 'none', 'random'
+
+
+    Returns
+    -------
+    :class:`Model`
+
+    References
+    ----------
+    https://arxiv.org/pdf/1905.11946.pdf
+
+    '''
+
+    def _make_divisible(v, divisor, min_value=None):
+        # make number of channel divisible
+        if min_value is None:
+            min_value = divisor
+        new_v = max(min_value, int(v + divisor / 2) // divisor * divisor)
+        # make sure that round down does not go down by more than 10%.
+        if new_v < 0.9 * v:
+            new_v += divisor
+        return new_v
+
+    def round_filters(filters, width_coefficient, depth_divisor):
+        # round number of filters based on width multiplier and depth_divisor
+
+        filters *= width_coefficient
+        new_filters = int(filters + depth_divisor / 2) // depth_divisor * depth_divisor
+        new_filters = max(depth_divisor, new_filters)
+        # Make sure that round down does not go down by more than 10%.
+        if new_filters < 0.9 * filters:
+            new_filters += depth_divisor
+        return int(new_filters)
+
+    def round_repeats(repeats, depth_coefficient):
+        # round number of repeats based on depth multiplier
+
+        return int(math.ceil(depth_coefficient * repeats))
+
+    def _MBConvBlock(inputs, in_channels, out_channels, ksize, stride, expansion, se_ratio, stage_id, block_id,
+                     noskip=False, activation_fn='relu'):
+        """
+        Inverted Residual Block
+
+        Parameters
+        ----------
+        inputs:
+            Input tensor
+        in_channels:
+            Specifies the number of input tensor's channel
+        expansion:
+            expansion factor always applied to the input size.
+        stride:
+            the strides of the convolution
+        alpha:
```

Comment:
I dont see any alpha parameter in the function

Reply:
fixed